src/pages/patterns/status-indicator-pattern/index.mdx

```diff
@@ -156,263 +236,161 @@ users, possibly hurting their overall experience.
 
-![Aligned status indicator icons.](images/status_indicator_5.png)
+![Aligned status indicator icons.](images/status-alignment-do.png)
 
   </DoDont>
-  <DoDont type="dont" colLg={6} caption="Do not push icons out of alignment with label length. In this case, where the status indicators are flush right, the two digit labels are pushing the icons out of alignment.">
+  <DoDont type="dont" caption="Do not push icons out of alignment with label length. In this case, where the status indicators are flush right, the two digit labels are pushing the icons out of alignment.">
 
-![Misaligned status indicator icons](images/status_indicator_6.png)
+![Misaligned status indicator icons](images/status-alignment-dont.png)
 
   </DoDont>
 </DoDontRow>
 
-### Badge indicators
-
-Badge indicators let the user know that something is new or updated. A badge
-status is displayed over a ghost icon button, usually in the header, to indicate
-an active notification and is cleared after the user acknowledges the
-notification. Depending on your use case, the icon button can open a new page or
-launch a modal, pane, or flyout.
-
-<Row>
-<Column colLg={8}>
-
-![Example of badge indicators in a global header](./images/status_indicator_7.png)
+## Shape indicator
 
-<Caption>
-  Badge statuses with numbers are usually used for global notifications.
-</Caption>
-
-</Column>
-</Row>
-
-#### Badge status with number
+Shape indicators use a shape, color, and descriptive label to convey the status
+of a device, feature, or version—without relying on a symbol.
 
-Numbers are used in conjunction with a badge status when a count of new or
-updated items is available and it's important for the user to know the number of
-updates.
-
-Badge status numbers can only be used in conjunction with the large icon button
-because with anything smaller, the icon gets covered. In very rare cases badge
-number may exceed two digits.
+Their use cases include helping users prioritize tasks, like a red triangle for
+high-priority tasks, or showing the current phase of a component's lifecycle,
+such as activated or deactivated. They don’t always necessarily indicate
+urgency. Shape indicators are also used in diagrams and are designed to remain
+clear even at small sizes.
 
 <Row>
 <Column colLg={8}>
 
-![Four icons with badges that included numbers ranging from one to three digits. The last badge has a plus.](./images/badge-status-with-number.jpg)
+![Example of a contextual shape status indicator](./images/status-shape-indicator.png)
+
+<Caption>Example of the status label directly next to the shape</Caption>
 
 </Column>
 </Row>
 
-#### Badge status without number
-
-A badge indicator with no number is used when a new notification is available
-and the number of notifications is either unknown or irrelevant to the user. The
-dot badge is less noticeable than the numbered badge, but still draws the user’s
-attention to the icon button.
-
-### Shape indicators
+### Shape indicator statuses
```

Comment:
Use a comma in this table instead of a slash when separating multiple color tokens like you do in the Icon indicator statuses table for consistency.